pkg/session/session.go

```diff
@@ -243,6 +314,10 @@ func newClient(ctx context.Context, logger logr.Logger, sessionKey string, url *
 			if err != nil {
 				logger.Error(err, "failed to keep alive govmomi client")
 				logger.Info("clearing the session")
+				sessionOperationMetric.With(prometheus.Labels{
+					metricLabelSessionKey:    sessionKey,
```

Comment:
This should use the server, dc, username pattern, not the session key as label?!